list2/ex44/index.js

```diff
@@ -0,0 +1,19 @@
+export function countingStringPropertiesInAnObject() {
+  const data = {
+    name: "Pedro",
+    age: 19,
+    city: "São Caetano do Sul",
+    email: "pedro.papa@cloudbeds.com",
+  };
+
+  let count = 0;
+  const stringCount = Object.entries(data);
+  stringCount.forEach(([_, value]) => {
+    let itIsString = typeof value === "string";
+    if (itIsString) {
+      count++;
+    }
+  });
```

Comment:
Your solution works as expected and there is nothing wrong with it but it could be simplified with the usage of Array methods such as filter or reduce. Here is an example with filter:

```js
const stringCount = Object.values(obj).filter(value => typeof value === "string").length;
```